_posts/2023-11-09-in-progress-wildfly-feature-on-openshift.adoc

```diff
@@ -0,0 +1,276 @@
+echo "" &&
+echo "rhsso Admin Console:   $RHSSO_URL/admin" &&
+echo ""
+```
+
+=== OpenShift Web Console
+Alternatively, you can click on the `Open URL` button on the pod to be redirected to the RH-SSO landing page and click on `Administration Console`. You will be able to sign in using the admin username and password that is specified in the yaml configuration of the RH-SSO pod. 
+
+== Configure RH-SSO
+
+First log in to the RH-SSO admin console using the username and password specified in the yaml configuration for the SSO pod.  From here, create a realm called `myrealm`, and a client called `myclient`. 
+
+After registering our client, `myclient`, we also need to configure `valid redirect URIs`. Once we have our application deployed on OpenShift, we will assign the URL from our application to the `Valid Redirect URIs` field. 
+
+Navigate to the `Client Scopes` tab for `myclient`. Here you will see that some scope values are under `Assigned Default Client Scopes` field and some others are under `Assigned Optional Client Scopes` field. When client scopes are added to `Default`, the claims associated with them are automatically added to the access token you receive without the need to configure them. Changing it to `Optional` ensures that they are not included by default and you will only get access to claims if you configure the corresponding scope values. This list of scope values you see here are the values that RH-SSO allows and it varies for different OpenID providers. RH-SSO also allows you to create new scopes, but we will not be doing that for this blog. 
+
+Now, click on `Roles` and create two roles, `user` and `admin`.
+
+Finally, create a user called `alice` whose `First Name` is Alice and `Last Name` is Smith and assign her the `user` and `admin` roles. Next, navigate to the `Credentials` tab and create a password for Alice. Toggle the `Temporary` switch off so you are not prompted to update the password after the first login. 
+
+== Build Container Images for WildFly on OpenShift
+
+Normally to use WildFly features that have been released, we just need to add the WildFly Helm release using the chart that is already available on OpenShift. However, since we are making use of a feature that is not released yet, we will need to build our own image of WildFly from the source code and deploy our application on that. 
+
+=== Clone the WildFly and Elytron Projects
+First we need to clone the `WildFly` and the `WildFly-Elytron` projects with the branches that contain the OIDC scope feature: 
+```
+git clone https://github.com/PrarthonaPaul/wildfly.git
+git checkout WFLY-16532
+```
+Next, go back to the directory that contains the `WildFly-Elytron` project and clone the `WildFly-Elytron` project: 
+```
+git clone https://github.com/PrarthonaPaul/wildfly-elytron.git
+git checkout WFLY-16532
+```
+Notice that we are not using `WildFly/WildFly` repo. Instead, we are using a forked version of the project. 
```

Comment:
Notice that we are not using `WildFly/WildFly` and wildfly-security/wildfly-elytron repositories. Instead, we are using a forked version of the projects.

Reply:
Fixed! 
Thanks for the comments!